facility-manager.html

```diff
@@ -0,0 +1,808 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="disk"></a>Disk usage</h2>
+
+<p>System administrators can view disk usage using the OMERO.web Admin tab. The disk usage reported reflects only the image data owned by the users or groups. Files that are uploaded, or created in situ, and attached to Projects, Datasets or Images, are not included in the disk usage statistics.</p>
+
+<ol>
+
+<li>
+<p class="first">Click on the <span class="bold">Statistics</span> tab to see the disk usage for all users and groups on the OMERO server.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerStats.jpg" class="align-center" src="images/facilityManagerStats.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="email"></a>Emailing users</h2>
+
+<p>The OMERO server can be configured so that administrators can use the OMERO.web Admin screen to email users through the OMERO system. Details on the configuration properties for mail are on <a href="https://www.openmicroscopy.org/site/support/omero/sysadmins/config.html#mail" target="_blank" class="external-url">Configuration properties glossary</a> in the sysadmin documentation.</p>
+
+<p>Recipients can be selected from all users, specific users or specific groups.</p>
+
+<ol>
+
+<li>
+<p class="first">Click on the <span class="bold">Email</span> tab to send emails to OMERO users or groups.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/sharingDataEmail.jpg" class="align-center" src="images/sharingDataEmail.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="upgrade"></a>Upgrading</h2>
+
+<p>OMERO and Bio-Formats are developed and released independently, so new releases of Bio-Formats that address bugs or enable new data type handling are not held up by OMERO development and vice versa.</p>
+
+<p>OMERO point releases, i.e. 5.2.x, occur regularly, and clients and servers are backwardly compatible for these. This is not the case for breaking releases, i.e. 5.x.</p>
```

Comment:
To be more specific a breaking release should always end up with 0 as the patch number `i.e. 5.x.0`.